src/components/BubbleModal.module.css

```diff
@@ -56,9 +57,15 @@
 
 .linkPreview {
   width: 100%;
-  height: 100%;
-  min-height: 200px;
+  height: 300px;
   background-color: #eee;
+  position: relative;
+}
+
+.linkPreviewImage {
+  width: 100%;
+  height: auto;
+  object-fit: cover;
```

Comment:
I think you forgot to fill out styles for `.linkPreviewImage`. I added some here.